install.md.hbs

```diff
@@ -273,14 +266,10 @@ this can reuse the `tap-registry` secret created in
 > **Note:** When you install Tanzu Application Platform, it is bootstrapped with
 > a set of dependencies (buildpacks and stacks) for application builds.
 > For more information about buildpacks, see the [VMware Tanzu Buildpacks Documentation](https://docs.vmware.com/en/VMware-Tanzu-Buildpacks/index.html).
-> You can find the buildpack and stack artifacts installed with Tanzu Application Platform
-> in the descriptor file on [Tanzu Network](https://network.pivotal.io/products/tbs-dependencies).
-> The current installed version of the descriptor is
-> [100.0.293](https://network.pivotal.io/products/tbs-dependencies#/releases/1086670). Sometimes the dependencies
-> require updates. You can use a
-> [manual process in a CI/CD context](tanzu-build-service/tbs-about.html#dependencies-manual), or
-> an [automatic update process](tanzu-build-service/tbs-about.html#auto-updates)
-> in the background by Tanzu Application Platform.
+> Sometimes the dependencies require updates. You can use a [manual process in a CI/CD context](tanzu-build-service/tbs-about.html#dependencies-manual), or
+> an experimental [automatic update process](tanzu-build-service/tbs-about.html#auto-updates) in the background by Tanzu Application Platform.
+> By default, a small-footprint (lite) set of dependencies is installed in TAP which does not contain all Buildpacks and Stacks.
```

Comment:
Maybe link to this other section you made?  As a reader looking at installing a profile, I wouldn't know which descriptor I should pick based on this description.  ### <a id="descriptor-comparison"></a> Descriptor comparison